test/Emu.Tests/Fixes/FrontierLabs/PartialFileRepairTests.cs

```diff
@@ -78,6 +78,21 @@ public PartialFileRepairTests(ITestOutputHelper output, FixtureHelper.FixtureDat
             // caused due to a crash on the device that failed to cleanup the end of the file
             // part of the file is encoded as flac - the rest is (?) WAVE data that is to be discarded
             new TestCase(PartialTestPartial0600, FixStatus.Fixed, "20220426T060000+1000_recovered.flac", 0, 2_281_472, 317292588, 2833099, "EMU+FL011").AsArray(),
+
+            // caused by a full-size preallocated file which has a WAVE header
+            // sensor crashed from
```

Comment:
what would you like to see explained

Reply:
i was meaning "sensor crashed from" just wondering because it looked like an incomplete sentence. but I think i understand now, it means"sensor crashed from": and the next line (the watchdog thing) is the explanation

Reply:
https://github.com/QutEcoacoustics/emu/commit/ecb2b27c5e3f8d1b80e33f7180e9aae8154bae80